CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Release 2.51.0 14-11-2024
+
+- Update Envoy to 1.30.7 (#936)
+- Update the docker build image in DEVELOPER.md to latest (#930)
+
```

Comment:
Remove this line  as this isn't user visible.

Reply:
line 4

Reply:
Done. PTAL!